logical_router_lb.go

```diff
@@ -0,0 +1,122 @@
+/**
```

Comment:
I think one file for operations for a table is enough, no need to add new files for handling each column.
(P.S. router LB support may be added separately, since it is not directly related to this commit.)

Reply:
I added because upstream doesnt have any table called Logical_Switch_Load_Balancer and Logical_Router_Load_Balancer. Hence, in lieu of that format, I created new file for load_balancer too. Is

Reply:
If you look at the current file structure, each file maps to a table. It is better to follow that. As you said there is no such table like Logical_Switch_Load_Balancer/Logical_Router_Load_Balancer, so you don't need a new file for that. You are operating Logical_Switch/Logical_Router table in this case, so just update those files.

Reply:
Cool. Sounds good!